server.ru

```diff
@@ -24,7 +24,12 @@ require 'action_dispatch'
 use Rack::Cors do
   allow do
     origins '*'
-    resource '*', headers: :any, methods: :any
+    resource '*',
+      headers: :any,
+      methods: :any,
+      expose: ['Cache-Control', 'Content-Encoding', 'Content-Type'],
+      max_age: 1728000,
+      credentials: false
```

Comment:
On the deleted file `credentials` and `methods` are different

```
  add_header 'Access-Control-Allow-Credentials' 'true' always;

Reply:
Yes it's because then gem asks to specify the origin when you set `credentials` to `true`. We don't need it in our case : https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Credentials